app/src/main/java/com/somestupidappproject/betago/utils/ScoreCount.java

```diff
@@ -0,0 +1,44 @@
+package com.somestupidappproject.betago.utils;
+
+/**
+ * Created by Sagar Patel on 7/24/2017.
+ */
+
+public class ScoreCount {
+    private int whiteScore;
+    private int blackScore;
+
+    public ScoreCount() {
+        this(0,0);
+    }
+
+    public ScoreCount(int initialWhiteScore, int initialBlackScore) {
+        whiteScore = initialWhiteScore;
+        blackScore = initialBlackScore;
+    }
+
+    public int getWhiteScore() {
```

Comment:
Refactor these getters and setters to take in a color. This will clean up code in ScoringUtils as well